framework/wazuh/core/stats.py

```diff
@@ -2,14 +2,207 @@
 # Created by Wazuh, Inc. <info@wazuh.com>.
 # This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
 
+import configparser
 import json
 import os
 from datetime import datetime
+from io import StringIO
 
-from wazuh.core import common
-from wazuh.core.exception import WazuhInternalError, WazuhError
+from wazuh.core import agent, common
+from wazuh.core.exception import (WazuhError, WazuhException,
+                                  WazuhInternalError, WazuhResourceNotFound)
 from wazuh.core.wazuh_socket import WazuhSocket
 
+DAYS = "Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat"
+MONTHS = "Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"
+
+
+def hourly_():
+    """Compute hourly averages
+
+    Returns
+    -------
+    array
+        averages and iterations
+    """
+    averages = []
+    interactions = 0
+    for i in range(25):
+        try:
+            hfile = open(common.stats_path + '/hourly-average/' + str(i))
```

Comment:
```suggestion
            hfile = open(f'{common.stats_path}/hourly-average/{str(i)}')
```

Review this kind of strings to use `f-strings`, as they are more efficient. This exact case could use `os.path.join` as well.